platform/ios/src/UIColor+MGLAdditions.mm

```diff
@@ -2,6 +2,12 @@
 
 @implementation UIColor (MGLAdditions)
 
+static inline unsigned ToByte(CGFloat f)
+{
+    f = MAX(0, MIN(f, 1)); // Clamp
```

Comment:
I think clamping the value between 0 and 1 would lead to inaccuracies on a wide color–enabled display, but then again I don’t know that mbgl configures OpenGL correctly for wide color. Probably it doesn’t, because the [`rgba`](https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/#rgba) expression is documented as accepting only values between 0 and 1.